src/event-mocks.js

```diff
@@ -51,8 +51,8 @@ function tablesStreams (eventName) {
         dynamodb: {
           ApproximateCreationDateTime: new Date() / 1000,
           // Keys + NewImage are in DynamoDB JSON, perhaps we can mock transform that later
-          Keys: { NOT_MOCKED: true },
-          NewImage: { NOT_MOCKED: true },
+          Keys: dynamoData.Keys,
+          NewImage: dynamoData.NewImage,
```

Comment:
Probably for these we should fall back to `{ NOT_MOCKED: true }` if the data is not present.